bench/tx-generator/src/Cardano/Benchmarking/Script/Env.hs

```diff
@@ -73,12 +77,21 @@ askIOManager = lift RWS.ask
 set :: Store v -> v -> ActionM ()
 set key val = lift $ RWS.modify $ (\e -> e { dmap = DMap.insert key (pure val) (dmap e)})
 
+setProtoParamMode :: ProtocolParameterMode -> ActionM ()
+setProtoParamMode val = lift $ RWS.modify $ (\e -> e { protoParams = pure val })
```

Comment:
Given that you'll have more of those cases, I'd split this into a combination of:

1. the generic `lift . RWS.modify` helper and have it accept the: 
2. the `Env` modifier for specific cases